x-pack/plugins/reporting/server/test_helpers/create_mock_reportingplugin.ts

```diff
@@ -36,30 +36,35 @@ const createMockSetupDeps = (setupMock?: any): ReportingSetupDeps => {
     licensing: {
       license$: of({ isAvailable: true, isActive: true, type: 'basic' }),
     } as any,
-    usageCollection: {} as any,
+    usageCollection: {
+      makeUsageCollector: jest.fn(),
+      registerCollector: jest.fn(),
+    } as any,
   };
 };
 
+export const createMockConfig = (overrides?: any) => ({
+  index: '.reporting',
+  kibanaServer: {
+    hostname: 'localhost',
+    port: '80',
+  },
+  capture: {
+    browser: {
+      chromium: {
+        disableSandbox: true,
+      },
+    },
+  },
+  ...overrides,
+});
```

Comment:
nit: since this doesn't return a `ReportingConfig` object, it may be better to have this be called `createMockConfigSchema`